README.md

```diff
@@ -10,6 +12,7 @@
 - A/B/n experiments
 - Mutually exclusive experiments (Layers)
 - Dynamic configuration
+- Multi-armed Contextual Bandits
```

Comment:
🔥 Don't want to forget that! Although I've always seen it phrased with "Contextual" first. `Contextual Multi-Armed Bandits`